platform/solr/platform-solr-server-standalone/src/main/resources/solr/conf/schema.xml

```diff
@@ -52,206 +49,169 @@
+       will increase storage costs.
+     required: The field is required.  It will throw an error if the
+       value does not exist
+     default: a value that should be used if no value is specified
+       when adding a document.
     -->
 
     <!-- field names should consist of alphanumeric or underscore characters only and
-       not start with a digit.  This is not currently strictly enforced,
-       but other field names will not have first class support from all components
-       and back compatibility is not guaranteed.  Names with both leading and
-       trailing underscores (e.g. _version_) are reserved.
+      not start with a digit.  This is not currently strictly enforced,
+      but other field names will not have first class support from all components
+      and back compatibility is not guaranteed.  Names with both leading and
+      trailing underscores (e.g. _version_) are reserved.
     -->
 
-    <!-- If you remove this field, you must _also_ disable the update log in solrconfig.xml
-       or Solr won't start. _version_ and update log are required for SolrCloud
-    -->
-    <field name="_version_" type="long" indexed="true" stored="true"/>
+    <!-- In this _default configset, only four fields are pre-declared:
+         id, _version_, and _text_ and _root_. All other fields will be type guessed and added via the
+         "add-unknown-fields-to-the-schema" update request processor chain declared in solrconfig.xml.
 
-    <!-- points to the root document of a block of nested documents. Required for nested
-       document support, may be removed otherwise
-    -->
-    <!--<field name="_root_"  type="string" indexed="true" stored="true" docValues="true"/>-->
+         Note that many dynamic fields are also defined - you can use them to specify a
+         field's type via field naming conventions - see below.
 
-    <field name="id_txt" type="string" indexed="true" stored="true" required="true"/>
-    <field name="id_txt_tokenized" type="text_general" indexed="true" stored="false"
-           required="false"/>
-    <field name="id_txt_tokenized_has_case" type="text_general_has_case" indexed="true"
-           stored="false" required="false"/>
-
-    <!-- Field to use to determine and enforce document uniqueness.
-     Unless this field is marked with required="false", it will be a required field
-  -->
-    <uniqueKey>id_txt</uniqueKey>
+         WARNING: The _text_ catch-all field will significantly increase your index size.
+         If you don't need it, consider removing it and the corresponding copyField directive.
+    -->
 
+    <!--<field name="id" type="string" indexed="true" stored="true" required="true" multiValued="false" />-->
+    <field name="id_txt" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true"/>
+    <field name="id_txt_tokenized" type="text_general" indexed="true" stored="false" required="false"/>
```

Comment:
❓ Do we need `id_txt_tokenized` ? Since we moved away from hyphenated uuid's this tokenization isn't applicable to our ID's even.

Reply:
There is a configuration to enable hyphenated UUIDs

Reply:
@bdeining Are some downstreams still using hyphenated?

However even if we do have hyphenated ids, how much does indexing those as separate terms benefit us? Is searching for partial bits of an ID a common use case? Even if we did want better searchability for the hyphenated ID case it seems like we'd be better served by splitting it and reconstructing it without hyphens in the index so they could search for either form (something like WordDelimiterGraphFilterFactory with `concatenateAll` set to 5)

Reply:
Yes.  But I'm just pointing it out.  Also, since the uuid generator service is an interface, one could write their own generator downstream that follows some other convention.